packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added a new SDK method (`ConfigUtils.hasTokenExpired`) that allows developers to verify whether a given JSON web token has expired. [#2298](https://github.com/zowe/zowe-cli/pull/2298)
```

Comment:
I think it'd be better to turn this into an action statement: Use the new SDK method `bbb` to check whether a given JSON web token has expired.
A note: Do you really use the token to verify expiration, as you originally wrote, or to check whether the token has expired?